.codecov.yml

```diff
@@ -3,20 +3,12 @@ coverage:
     round: down
     range: "70...100"
 
-    status:
-        project:
-            default: on
-        patch:
-            default: off
-        changes:
-            default: off
-
```

Comment:
`default: on` is invalid.  Removing this appears to have made it so that as long as coverage is going up the check will be green.